### PR TITLE
Read Chungus.

### PR DESCRIPTION
**Pull request by Jacoby6000**
Testing the modified setup script.  It should be more user friendly now.  Also there's a readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,121 @@
...
+#### Automatic
+
+Currently automatic installation is supported only in Ubuntu, and only for running via `systemd`.  If you do not want to
+use systemd, or you are not on ubuntu, perform the [manual](#manual) installation.
+
+If you are on ubuntu, and you do wish to use systemd, the following should be sufficient for downloading and installing 
+the latest version of BigOlChungus.
+
+```bash
+wget https://raw.githubusercontent.com/Jacoby6000/bigolchungus/feature/readme/setup-miner.sh && \
```
Comment (sir-wabbit) [inline README.md]: wrong link

Comment (Jacoby6000) [reply README.md]: That's intentional atm.  I'm testing to make sure all of the stuff is copy/pastable and that the setup script works.

Comment (Jacoby6000) [reply README.md]: I'll be sure to change it back

**Comment by Jacoby6000** [comment]
Setup script output: https://hastebin.com/ukenifuxoq.rb


Notable things:

The chainweb release is gone?
```
Installing chanweb-miner...
--2019-11-14 03:23:43--  https://github.com/kadena-io/chainweb-node/releases/download/1.0.2/chainweb.8.6.5.ubuntu-18.04.fdf9d0e3.tar
Resolving github.com (github.com)... 140.82.113.4
Connecting to github.com (github.com)|140.82.113.4|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-11-14 03:23:44 ERROR 404: Not Found.
tar (child): chainweb.8.6.5.ubuntu-18.04.fdf9d0e3.tar: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

ubuntu-drivers doesn't exist (on 18.04 LTS in google app engine)
```
Installing OpenCL...
./setup-miner.sh: 69: ./setup-miner.sh: ubuntu-drivers: not found
```

**Comment by Jacoby6000** [comment]
False alarm. I was using Debian by mistake.

**Comment by Jacoby6000** [comment]
Seems to all be good. Need to wait for a `chainweb-miner` binary to be published, though.

https://github.com/kadena-io/chainweb-miner/issues/2

File: CONTIRBUTING.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Embrace the chungus.
```
Comment (fosskers) [inline CONTIRBUTING.md]: Spelling of the filename ;)

**Review by fosskers — CHANGES_REQUESTED** [review]

File: CONTIRBUTING.md (hunk 2 above)
Comment (Jacoby6000) [reply CONTIRBUTING.md]: lmao. "contirbuting"

**Comment by Jacoby6000** [comment]
This is ready. I tested the steps in the guide end-end with the updated setup script using a GAE instance with Ubuntu 18.04 Minimal LTS and had no hiccups.

**Pull request merged**